examples/sdk-examples/src/examples/chartResponsiveness/index.tsx

```diff
@@ -5,19 +5,28 @@ import React from "react";
 import { ExampleWithSource } from "../../components/ExampleWithSource";
 import { HeadlineResponsiveExample } from "./HeadlineResponsiveExample";
 import { BarChartResponsiveExample } from "./BarChartResponsiveExample";
+import { ColumnChartResponsiveLegendExample } from "./ColumnChartResponsiveLegendExample";
 
 import HeadlineResponsiveExampleSrc from "!raw-loader!./HeadlineResponsiveExample";
 import BarChartResponsiveExampleSrc from "!raw-loader!./BarChartResponsiveExample";
+import ColumnChartResponsiveLegendExampleSrc from "!raw-loader!./ColumnChartResponsiveLegendExample";
 
 import HeadlineResponsiveExampleSrcJS from "!raw-loader!../../../examplesJS/chartResponsiveness/HeadlineResponsiveExample";
 import BarChartResponsiveExampleSrcJS from "!raw-loader!../../../examplesJS/chartResponsiveness/BarChartResponsiveExample";
+import ColumnChartResponsiveLegendExampleSrcJS from "!raw-loader!../../../examplesJS/chartResponsiveness/ColumnChartResponsiveLegendExample";
 
 export const ChartResponsiveness: React.FC = () => (
     <div>
         <h1>Chart Responsiveness</h1>
-        <p>These examples show improved responsive behaviour of charts.</p>
+        <p>These examples show improved responsive behaviour of charts and headline.</p>
 
         <h2 id="headline">Headline</h2>
+        <p>
+            Headline shows pagination for smallest size. If there is enough vertical space, it displays
+            additional values below each other. When there is enough horizontal space, it uses the default
+            side-by-side layout.
```

Comment:
When rendered in the smallest possible size, headlines enable pagination. If there is enough vertical space, a headline displays additional values one under the other. If there is enough horizontal space, it uses the default side-by-side layout.